src/renderer/views/Channel/Channel.vue

```diff
@@ -341,6 +350,15 @@ const isElementListLoading = ref(false)
 const isSearchTabLoading = ref(false)
 const currentTab = ref('videos')
 
+const hideWatchedEntries = ref(false)
+
+/**
+ * @param {boolean} value
+ */
+function updatehideWatchedEntries(value) {
```

Comment:
Find and replace mistake made with case insensitive mode I can see :)
```suggestion
function updateHideWatchedEntries(value) {
```

Reply:
Good eye, I even replaced entries one by one to try to avoid stuff like that and it still escaped me. Just fixed it, thank you very much